solr/solrj/src/java/org/apache/solr/client/solrj/response/QueryResponse.java

```diff
@@ -184,6 +187,15 @@ else if ( "terms".equals( n ) ) {
       else if ( "moreLikeThis".equals( n ) ) {
         _moreLikeThisInfo = (NamedList<SolrDocumentList>) res.getVal( i );
       }
+      else if ("taskList".equals( n )) {
```

Comment:
Do we need three separate sections for these? I don't think it's possible to return all three in the same call, it's either this or that, so one section eg. `taskInfo` should be enough?

Reply:
I didnt quite parse that -- you mean, reuse `taskInfo` across? How would that work, since each of the three calls returns a different type?